.github/workflows/pre_merge.yaml

```diff
@@ -68,21 +68,30 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - python-version: "3.10"
-            tox-env: "py310"
-    name: Integration-Test-Py${{ matrix.python-version }}
+          - task: "MULTI_CLASS_CLS"
+          - task: "MULTI_LABEL_CLS"
+          - task: "H_LABEL_CLS"
+          - task: "DETECTION"
+          - task: "ROTATED_DETECTION"
+          - task: "INSTANCE_SEGMENTATION"
+          - task: "SEMANTIC_SEGMENTATION"
+          - task: "ACTION_CLASSIFICATION"
+          - task: "ACTION_DETECTION"
+          - task: "VISUAL_PROMPTING"
+          - task: "ZERO_SHOT_VISUAL_PROMPTING"
```

Comment:
I personally think that it is too granularly divided. Each matrix should install the dependency (torch, ...) and it seems too costly for me. Can we group them to the bigger ones?

Reply:
https://github.com/openvinotoolkit/training_extensions/pull/2914/commits/41688b8ecf13f43fcf9981eb32f06a2a81acbfc3
I grouped them into the following categories: classification, detection, action, instance_segmentation, semantic_segmentation, visual_prompting.